core/src/main/java/org/kiwix/kiwixmobile/core/utils/files/FileUtils.kt

```diff
@@ -310,16 +311,14 @@ object FileUtils {
     return null
   }
 
-  /**
```

Comment:
Why have you removed the comment about what does the function? It should not IMO.

Reply:
@kelson42 ,
this documentation is outdated. we are using a new function ```URLUtil.guessFileName()``` of ```android.webkit``` and the updated documentation is available in that class.

Reply:
If documentation is outdated, you updated it! You don't remove it. Reminder: Such a comment is there to explain:
* The added value (from a user perspective) of this function
* The principle of functionning
* The exception handling
* Anything which is not obvious and necessary to know by using this function

Reply:
i have added documentation for this function . also updated the PR description.